lib/widgets/app_drawer.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+
+import '../screens/orders_screen.dart';
+
+class AppDrawer extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Drawer(
+      child: Column(
+        children: [
+          AppBar(
+            title: Text('Hello Friend!'),
+            automaticallyImplyLeading: false,
+          ),
+          Divider(),
+          ListTile(
+            leading: Icon(Icons.shop),
+            title: Text('Shop'),
+            onTap: () => Navigator.of(context).pushReplacementNamed('/'),
```

Comment:
would be nice if this is distinguished like other routes so that it would be easier to understand which page will be shown